src/main/java/com/github/black0nion/blackonionbot/systems/music/PlayerManager.java

```diff
@@ -81,17 +82,17 @@ public GuildMusicManager getMusicManager(final TextChannel channel) {
 		});
 	}
 
-	public void loadAndPlay(final BlackUser author, final TextChannel channel, String trackUrl, final AudioManager manager, final AudioChannel vc) {
+	public void loadAndPlay(final BlackUser author, final TextChannel channel, @NotNull String trackUrl, final AudioManager manager, final AudioChannel vc) {
```

Comment:
I personally decided to use @Nonnull instead of NotNull, please change this if you're fine with that choice.